ci/test-conformance-eks.sh

```diff
@@ -46,6 +48,8 @@ Setup a EKS cluster to run K8s e2e community tests (Conformance & Network Policy
         --k8s-version            GKE K8s cluster version. Defaults to 1.17.
         --aws-access-key         AWS Acess Key for logging in to awscli.
         --aws-secret-key         AWS Secret Key for logging in to awscli.
+        --aws-service-user-arn   AWS Service User Role Name for logging in to awscli.
```

Comment:
Same as the previous comment on parameter names.

The help string should be `AWS Service User Role ARN for logging in to awscli` (change "Name" to "ARN"). This is because ARN is like an ID, not a name.